File: app/src/at-trinhnguyen/java/asiantech/internship/winter/javacore/oop/Square.java
package asiantech.internship.winter.javacore.oop;

public class Square extends Shape {
    private double edge;

    public Square(double edge) {
        this.edge = edge;
    }


    @Override
    public double area() {
        return edge * edge;
    }

    @Override
    public double circumference() {
        return edge * 4;
    }

    @Override
    public void print() {
        System.out.print("Square ");
    }
}

File: app/src/at-trinhnguyen/java/asiantech/internship/winter/javacore/oop/Square.kt
package asiantech.internship.winter.javacore.oop

class Square(private val edge: Double) : Shape() {


    public override fun area(): Double {
        return edge * edge
    }

    public override fun circumference(): Double {
        return edge * 4
    }

    public override fun print() {
        print("Square ")
    }
}
